uwsgiconfig.py

```diff
@@ -27,7 +27,7 @@
 except ImportError:
     import configparser as ConfigParser
 
-
+PLUGINDIR_BASE = os.environ.get('PLUGIN_DIR_BASE', '')
```

Comment:
commit mesage say PLUGIN_BASE_DIR, code use PLUGIN_DIR_BASE and variable name is PLUGINDIR_BASE :) Can we use PLUGIN_BASE_DIR everywhere please?

Reply:
Ah! Yeah, of course. It was a long day. I will change that now.

Reply:
And done. I also rebased to make the commit history a bit cleaner.